diff --git a/makefile b/makefile
index 243c450..8f7dd8b 100644
--- a/makefile
+++ b/makefile
@@ -88,8 +88,6 @@ endif
 
 SHELL:=/usr/bin/env bash
 
-GITHASH = bin/githash.txt
-
 
 $(OBJ_DIR)/$(DC)/%.o: %.d
 	${DC} ${DC_FLAGS} ${VERSIONS} ${INCLUDE_PATHS} -c $< ${WRITE_TO_TARGET_NAME}
@@ -97,7 +95,7 @@ $(OBJ_DIR)/$(DC)/%.o: %.d
 $(UT_OBJ_DIR)/$(DC)/%.o: %.d
 	${DC} ${DC_TEST_FLAGS} ${VERSIONS} ${INCLUDE_PATHS} -c $< ${WRITE_TO_TARGET_NAME}
 
-${DSCANNER_BIN}: ${GITHASH} ${OBJ_BY_DC} | ${DSCANNER_BIN_DIR}
+${DSCANNER_BIN}: ${OBJ_BY_DC} | ${DSCANNER_BIN_DIR}
 	${DC} ${OBJ_BY_DC} ${WRITE_TO_TARGET_NAME}
 
 ${OBJ_BY_DC}: | ${OBJ_BY_DC_DIR}
@@ -120,12 +118,7 @@ all: ${DSCANNER_BIN}
 ldc: ${DSCANNER_BIN}
 gdc: ${DSCANNER_BIN}
 
-githash: ${GITHASH}
-
-${GITHASH}:
-	mkdir -p bin && ${GIT} describe --tags --always > ${GITHASH}
-
-debug: ${GITHASH}
+debug:
 	${DC} -w -g -Jbin -ofdsc ${VERSIONS} ${DEBUG_VERSIONS} ${INCLUDE_PATHS} ${SRC}
 
 # compile the dependencies separately, s.t. their unittests don't get executed
@@ -134,7 +127,7 @@ ${UT_DSCANNER_LIB}: ${LIB_SRC} | ${UT_DSCANNER_LIB_DIR}
 
 test: ${UT_DSCANNER_BIN}
 
-${UT_DSCANNER_BIN}: ${UT_DSCANNER_LIB} ${GITHASH} ${UT_OBJ_BY_DC} | ${DSCANNER_BIN_DIR}
+${UT_DSCANNER_BIN}: ${UT_DSCANNER_LIB} ${UT_OBJ_BY_DC} | ${DSCANNER_BIN_DIR}
 	${DC} ${UT_DSCANNER_LIB} ${UT_OBJ_BY_DC} ${WRITE_TO_TARGET_NAME}
 	./${UT_DSCANNER_BIN}
 
