diff --git a/common/dub.sdl b/common/dub.sdl
index 57b7c42..a995488 100644
--- a/common/dub.sdl
+++ b/common/dub.sdl
@@ -1,6 +1,4 @@
 name "common"
-stringImportPaths "../bin"
-preBuildCommands "\"$DC\" -run \"$PACKAGE_DIR/dubhash.d\""
 
 sourcePaths "src"
 importPaths "src"
diff --git a/common/src/dcd/common/dcd_version.d b/common/src/dcd/common/dcd_version.d
index 2d4cccb..f06c68b 100644
--- a/common/src/dcd/common/dcd_version.d
+++ b/common/src/dcd/common/dcd_version.d
@@ -24,14 +24,4 @@ import std.string : strip;
  * Human-readable version number
  */
 
-version (built_with_dub)
-{
-	enum DCD_VERSION = import("dubhash.txt").strip;
-}
-else
-{
-	/**
-	 * Current build's Git commit hash
-	 */
-	enum DCD_VERSION = import("githash.txt").strip;
-}
+enum DCD_VERSION = "nix_version";
diff --git a/makefile b/makefile
index 74c0b57..82ca488 100644
--- a/makefile
+++ b/makefile
@@ -15,10 +15,6 @@ DSYMBOL_DIR := dsymbol
 
 SHELL:=/bin/bash
 
-githash:
-	@mkdir -p bin
-	git describe --tags > bin/githash.txt
-
 report:
 	dscanner --report src > dscanner-report.json
 	sonar-runner
@@ -110,30 +106,30 @@ override DMD_SERVER_FLAGS += $(DFLAGS)
 override LDC_SERVER_FLAGS += $(DFLAGS)
 override GDC_SERVER_FLAGS += $(DFLAGS)
 
-dmdclient: githash
+dmdclient:
 	mkdir -p bin
 	${DMD} ${CLIENT_SRC} ${DMD_CLIENT_FLAGS}
 
-dmdserver: githash
+dmdserver:
 	mkdir -p bin
 	${DMD} ${SERVER_SRC} ${DMD_SERVER_FLAGS}
 
-debugserver: githash
+debugserver:
 	mkdir -p bin
 	${DMD} ${SERVER_SRC} ${DEBUG_SERVER_FLAGS}
 
-gdcclient: githash
+gdcclient:
 	mkdir -p bin
 	${GDC} ${CLIENT_SRC} ${GDC_CLIENT_FLAGS}
 
-gdcserver: githash
+gdcserver:
 	mkdir -p bin
 	${GDC} ${SERVER_SRC} ${GDC_SERVER_FLAGS}
 
-ldcclient: githash
+ldcclient:
 	${LDC} ${CLIENT_SRC} ${LDC_CLIENT_FLAGS} -oq -of=bin/dcd-client
 
-ldcserver: githash
+ldcserver:
 	${LDC} $(LDC_SERVER_FLAGS) ${SERVER_SRC} -oq -of=bin/dcd-server
 
 test: debugserver dmdclient
